app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PagesController < ApplicationController
+  def home
+    @products = Product.all
+    
+    @products = @products.where(category: :entree) if params[:category] == "entree"
```

Comment:
I think a `case when` would've worked better here

Reply:
Because the params you send are the same with the enum values, you could also use directly:
```ruby
@products.where(category: params[:category])
```